sdk/core/azure-core/test/ut/context.cpp

```diff
@@ -83,7 +83,7 @@ TEST(Context, BasicChar)
 
 TEST(Context, ApplicationContext)
 {
-  Context appContext = GetApplicationContext();
+  Context appContext;
```

Comment:
Looks like the intention of this test was specifically to test `ApplicationContext` from the test name, which we are no longer testing.

How are we supposed to test `ApplicationContext` (along with cancelling it)?

Maybe all other tests should avoid using a global context. To keep the test at least somewhat related to ApplicationContext, change `appContext` to be a child of the root ApplicationContext.

Reply:
Application context is not different than any Context.
It would be like testing a global std::string instead of any std::string specifically.

We call GetAppContext for multiple places.  If we really want to have such a test (useless), we would want to create a unique test exe just for that test so it never affect the other tests.